### PR TITLE
Add "Web Messages" field group to support AJO inbound delivery experience 

### PR DESCRIPTION
**Pull request by krishnanmb**
Issue: #1407

This PR creates a new standard field group "Web Messages", intending to extend profile schema,
to support AJO inbound delivery support.

### CODE REVIEW

File: extensions/adobe/experience/customerJourneyManagement/profile-web-messages.example.1.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "https://ns.adobe.com/experience/customerJourneyManagement/webMessages": {
+    "navBar": {
+      "journeyUUID_1": {
+        "messageId": "MessageUUID_1",
+        "messageExecutionId": "MessageExecutionUUID_5",
+        "messageVersionId": "MessageVersionUUID_3",
+        "journeyVersionId": "JourneyVersionUUID_1"
+      },
+      "journeyUUID_2": {
```
Comment (mjasrotia) [inline extensions/adobe/experience/customerJourneyManagement/profile-web-messages.example.1.json:10]: "_" is not allowed in AEP property names

**Review by cmathis — APPROVED** [review]

**Review by mjasrotia — APPROVED** [review]

**Comment by donwalling** [comment]
@krishnanmb @cmathis we need to hold on this getting into production. There are still open questions about technical strategy that need to be addressed.

**Comment by prabhum2** [comment]
@donwalling - reverting is the best option , We can always create a new PR when your team is ready. Created a new PR #1416 for the same . Please let me know if i can go ahead and revert the merge.

**Comment by donwalling** [comment]
thank you for merging the revert PR

**Pull request merged**